include/bl32/payloads/tlk.h

```diff
@@ -36,18 +36,22 @@
 /*
  * Generate function IDs for the Trusted OS/Apps
  */
+#if !ERROR_DEPRECATED
 #define TLK_TOS_STD_FID(fid)	((fid) | 0x72000000 | (0 << 31))
 #define TLK_TA_STD_FID(fid)	((fid) | 0x70000000 | (0 << 31))
+#endif
```

Comment:
@vwadekar 
As per @danh-arm comment, is it ok to *not* have this migration path and simply remove the current STD-based APIs?